sklearn/preprocessing/tests/test_data.py

```diff
@@ -1667,6 +1667,26 @@ def test_robust_scaler_unit_variance():
     assert X_trans.std() == pytest.approx(1, abs=1e-2)
 
 
+@pytest.mark.parametrize(
+    "array_namespace, device, dtype", yield_namespace_device_dtype_combinations()
+)
+@pytest.mark.parametrize(
+    "check",
+    [check_array_api_input_and_values],
+    ids=_get_check_estimator_ids,
+)
+@pytest.mark.parametrize(
+    "estimator",
+    [MaxAbsScaler()],
+    ids=_get_check_estimator_ids,
+)
+def test_maxabscaler_array_api_compliance(
```

Comment:
We could probably merge this test with `test_minmaxscaler_array_api_compliance` via the `estimator` parametrization and rename the test to `test_scaler_array_api_compliance` instead of introducing a new test each time for each scaler.